br/tests/lightning_character_sets/run.sh

```diff
@@ -75,3 +75,17 @@ run_lightning --config "tests/$TEST_NAME/binary.toml" -d "tests/$TEST_NAME/mixed
 run_sql 'SELECT sum(`唯一键`) AS s FROM charsets.mixed'
 check_contains 's: 5291'
 
+# test about unsupported charset in UTF-8 encoding dump files
+# test local backend
+run_lightning --config "tests/$TEST_NAME/greek.toml" -d "tests/$TEST_NAME/greek" 2>&1 | grep -q "Unknown character set: 'greek'"
```

Comment:
Can this run succeed?

Reply:
lightning will meet "Unknown character set: 'greek'" error, and grep will act as a negative test. CI has passed.

After user manually create the database and table, this PR will let lightning ignore the error which is checked in line 84, 89.